internal/core/services/events_forwarder/events_forwarder_service.go

```diff
@@ -240,6 +244,9 @@ func (es *EventsForwarderService) getPlayerInfo(event *events.Event) (string, er
 		return "", fmt.Errorf("playerId not found on eventAttributes")
 	}
 
-	playerId := event.Attributes["playerId"].(string)
+	playerId, ok := event.Attributes["playerId"].(string)
```

Comment:
It could panic here also, we always need to make a safe check when making typecast or checking map values